client.bal

```diff
@@ -106,7 +105,7 @@ public client class Client {
     #
     # + account - Account type record with detail
     # + return - If success returns a RecordAddResponse type record otherwise the relevant error
-    @display{label: "Add new account"}
+    @display{label: "Add a new account record"}
```

Comment:
I would say `Add New Account` is better. Shall we update the display annotation accordingly?

Reply:
Resolved. Removed the articles too.